src/app/contextBroker/contextBroker.cpp

```diff
@@ -1771,14 +1763,14 @@ int main(int argC, char* argV[])
     LM_T(LmtHttps, ("httpsKeyFile:  '%s'", httpsKeyFile));
     LM_T(LmtHttps, ("httpsCertFile: '%s'", httpsCertFile));
 
-    restInit(rsP, ipVersion, bindAddress, port, mtenant, connectionMemory, maxConnections, reqPoolSize, rushHost, rushPort, allowedOrigin, httpsPrivateServerKey, httpsCertificate);
```

Comment:
Yeah, I was only referring to changing the numerical values for defiinitions, not removing the parameters themselves:
The changes would be made in rest.h:

    extern void restInit
    (
       RestService*        _restServiceV,
       IpVersion           _ipVersion,
       const char*         _bindAddress,
       unsigned short      _port,
       bool                _multitenant       = false,
       unsigned int        _connectionMemory  = 64,
       unsigned int        _maxConnections    = 128,
       unsigned int        _mhdThreadPoolSize = 128,
       const std::string&  _rushHost          = "",
       unsigned short      _rushPort          = 0,
       const char*         _allowedOrigin     = NULL,
       const char*         _httpsKey          = NULL,
       const char*         _httpsCert         = NULL,
       RestServeFunction   _serveFunction     = NULL,
       bool                _acceptTextXml     = false
    );

The 64 and 128:s could be definitions in limits.h instead of 'plain numbers'.